### PR TITLE
Add clickXY which lets you click arbitrary coords.

### PR DESCRIPTION
**Pull request by joeheyming**
I found a stackoverflow which suggested you can run
document.elementFromPoint(x,y) to get an element.
This followed by click() should do the right thing.

I added a unit test just to prove it.

### CODE REVIEW

**Comment by angrykoala** [comment]
Hi, thanks for your PR everything looks fine, however I would like your opinion of what do you think about merging clickXY into click?

something like:
```js
browser.click("#my-id", 1) // click given element
browser.click(2,2); // clicks coorditantes
```

Because a selector will never be a Number, we could modify the behaviour (clicking a private method _clickXY) if click is called with numbers

I feel this may be a better approach for end user, what do you think?

**Comment by joeheyming** [comment]
I like to keep my method signatures clean.  But I'm willing to humor the other approach

File: lib/mixins/browser_actions.js [hunk 1]
```diff
@@ -75,8 +75,19 @@ module.exports = function BrowserActionsMixin(s) {
             });
         }
 
+        /**
```
Comment (angrykoala) [inline lib/mixins/browser_actions.js]: I'll document this in the readme so no need to add a comment

**Comment by angrykoala** [comment]
Thanks @joeheyming , the CI failed due to a server error, I restarted it, as soon as it finish I'll merge this.

**Comment by angrykoala** [comment]
I manually merged this to the `dev` branch instead of master so I'll close this PR as it doesn't detect it as merged

thanks for your help

**Pull request closed without merging**